doc/samtools-flagstat.1

```diff
@@ -52,7 +52,8 @@ Does a full pass through the input file to calculate and print statistics
 to stdout.
 
 Provides counts for each of 13 categories based primarily on bit flags in
-the FLAG field.
+the FLAG field. Information on the meaning of the flags is given in the SAM specification
+document (https://samtools.github.io/hts-specs/SAMv1.pdf).
```

Comment:
The paragraph break has gone missing here.  I'll fix on merge.

I'll also change the brackets to angle ones `< >` for consistency with other URLs referenced in SAMtools man pages.